p2panda-auth/src/group/mod.rs

```diff
@@ -0,0 +1,752 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#![allow(clippy::type_complexity)]
+#![allow(dead_code)]
```

Comment:
Clippy attributes should be local and maybe even with a comment to avoid overseeing linting errors.

I suggest taking them all away first, checking if they're still necessary, most stuff is maybe even used already, otherwise consider exporting on crate level. If that still doesn't work, it's fine to apply the rule, but better on local level.